### PR TITLE
docs: correct the security reporting email addresses

### PR DESCRIPTION
**Pull request by nedbat**
**Merge checklist:**
Check off if complete *or* not applicable:
- [ ] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Fixup commits are squashed away
- [ ] Unit tests added/updated
- [ ] Manual testing instructions provided
- [ ] Noted any: Concerns, dependencies, deadlines, tickets

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,15 @@ Change Log
    This file loosely adheres to the structure of https://keepachangelog.com/,
    but in reStructuredText instead of Markdown.
 
+2023-06-06
+**********
+
+Fixed
+=====
+
+- Corrected all security email address to security@axim.org.
```
Comment (timmc-edx) [inline CHANGELOG.rst:14]: I believe we should be using security@openedx.org instead.

Comment (nedbat) [reply CHANGELOG.rst:14]: This is the email address on https://openedx.org/community/connect/, but I'll use whatever is the right address.

Comment (timmc-edx) [reply CHANGELOG.rst:14]: Here's an issue on the wg-security issue tracker: https://github.com/openedx/wg-security/issues/15

They still have a lot of places to update, and that page is one of them, unfortunately.

Comment (nedbat) [reply CHANGELOG.rst:14]: @feanil ?  What is the canonical address to use?

Comment (nedbat) [reply CHANGELOG.rst]: OK, I changed it to openedx.org

**Pull request merged**